src/chromium/crBrowser.ts

```diff
@@ -50,24 +50,35 @@ export class CRBrowser extends BrowserBase {
     const connection = new CRConnection(SlowMoTransport.wrap(transport, slowMo));
     const browser = new CRBrowser(connection);
     const session = connection.rootSession;
-    const promises = [
-      session.send('Target.setDiscoverTargets', { discover: true }),
```

Comment:
where did this go?

Reply:
We don't use it except for initial target discovery (before auto attach is on).